### PR TITLE
added a check on Quick Start and Doc links which are present in the cards in "Enabled" page of dashboard

### PR DESCRIPTION
**Pull request by pranavsuryaw**
Signed-off-by: Pranav Suryawanshi <prsuryaw@redhat.com>

### CODE REVIEW

File: tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot [hunk 1]
```diff
@@ -593,3 +593,13 @@ RHODS Notification Drawer Should Not Contain
...
+
+Check Doc And Quick Start Link In Enabled Page Tile
+    [Documentation]    Checks for the Doc and Quick Start Link in the tile of enabled page
+    ...                ${id} is the id of the tile
+    [Arguments]    ${id}
+    Click Button    //article[@id="${id}"]/div/div/div/button
+    ${href} =    Get Element Attribute    //a[contains(text(),"View documentation")]    href
+    Check HTTP Status Code    ${href}
+    Click Element    //a[contains(text(),"Open quick start")]
+    Wait Until Page Contains    Start
```
Comment (bdattoma) [inline tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot:605]: could you create a kw in `ODHDashboardResources.resource` like `Page Should Contain QuickStart Sidebar` and use it here? 
You can find a similar kw in the same resource file, called `Page Should Not Contain QuickStart Sidebar`.

Besides, could you also close the quick start? You'll find keywords to handle it in above mentioned resource file

**Review by bdattoma — CHANGES_REQUESTED** [review]
could you add the same keyword to RHOAM test case and run test the PR on Jenkins? (you could exclude RHOAM from the test)

File: tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot [hunk 2]
```diff
@@ -593,3 +594,14 @@ RHODS Notification Drawer Should Not Contain
...
     Page Should Not Contain  text=${message}
     Click Element    xpath=//*[contains(@class,'notification-badge')]
+
+Check Doc And Quick Start Link In Enabled Page Tile
+    [Documentation]    Checks for the Doc and Quick Start Link in the tile of enabled page
+    ...                ${id} is the id of the tile
+    [Arguments]    ${id}
+    Click Button    //article[@id="${id}"]/div/div/div/button
+    ${href} =    Get Element Attribute    //a[contains(text(),"View documentation")]    href
+    Check HTTP Status Code    ${href}
```
Comment (bdattoma) [inline tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot:604]: could you also click on the Doc link? to check if the link is clickable

File: tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot [hunk 3]
```diff
@@ -593,3 +594,14 @@ RHODS Notification Drawer Should Not Contain
...
+    Click Element    //a[contains(text(),"Open quick start")]
+    Page Should Contain QuickStart Sidebar
+    Click Button    //button[@aria-label="Close drawer panel"]
```
Comment (bdattoma) [inline tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot:607]: try using `Close QuickStart From Button` or `Close QuickStart From Top`

Comment (pranavsuryaw) [reply tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot]: Done

**Comment by pranavsuryaw** [comment]
https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1007/console

**Comment by bdattoma** [comment]
> https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1007/console

I see failures in the run. The anaconda one failed running the kw implemented in this PR. Could you look into it?

**Comment by pranavsuryaw** [comment]
> > https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1007/console
> 
> I see failures in the run. The anaconda one failed running the kw implemented in this PR. Could you look into it?
3 of them passed the test
aikit failed at teardown but [it has passed my code]
for anaconda, I tested it again it failed but has passed my lines of code

**Comment by pranavsuryaw** [comment]
https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1025/console

File: tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot (hunk 2 above)
Comment (jgarciao) [reply tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot:612]: You can click to check if the link works and then close the window to go back

**Comment by pranavsuryaw** [comment]
https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1038/console

**Review by bdattoma — APPROVED** [review]

**Comment by bdattoma** [comment]
too old PR. If needed we can re-use it in the future, but there is no near term plan to implement this TC

**Pull request closed without merging**